content/metadata-rework.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Metadata rework"
```

Comment:
I looked at previous posts, e.g., https://stratis-storage.github.io/thin-provisioning-redesign/ . The title for that one is not very strongly stated, either. The consequence for the storage stack is an important part of what is going on and I wish it could be included in the title somehow. "Introduction to Stratis's V2 metadata format and storage stack"?